### PR TITLE
feat: login-signup page

### PR DESCRIPTION
**Pull request by ShekharDhangar**
Added login-signup page. Click on profile icon for login page . Please review

### CODE REVIEW

**Review by ParthParolekar — APPROVED** [review]
Everything Looks great. The UI is clean too.

File: authentication/login.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+const signUpForm = document.getElementById('sign-up'),
+trylogIn = document.getElementById('try-login'),
+trySignUp = document.getElementById('try-signup'),
+logInForm = document.getElementById('login');
+
+trySignUp.addEventListener('click', changeForm)
+trylogIn.addEventListener('click', changeForm)
+ 
+function changeForm(){
```
Comment (vishal804) [inline authentication/login.js:13]: You Can Use ES 6  Standard  It will Help you to Understand syntax

Comment (vishal804) [reply authentication/login.js:13]: Everything Else looks Good

**Review by vishal804 — APPROVED** [review]

File: authentication/login.js (hunk 1 above)
Comment (ShekharDhangar) [reply authentication/login.js:13]: Made the changes

**Pull request merged**